packages/client/src/utils/validation/validators.ts

```diff
@@ -16,3 +16,20 @@ export const validateRepeatedPassword = (
     ? ERRORS.notEquals
     : undefined
 }
+export const validateNewPassword = (password: string, values: FieldValues) => {
+  return validatePassword(password)
+    ? validatePassword(password)
+    : password === values.old_password
+    ? ERRORS.equalsPassword
+    : undefined
+}
+export const validateRepeatedNewPassword = (
```

Comment:
Чтобы не дублировать функции, лучше сделать так, чтобы функция принимала две строки и в конфиге сразу передавать из values нужное значение
```suggestion
export const validateRepeatedNewPassword = (password: string, secondPassword: string) => {
```